pkg/ocm/helpers.go

```diff
@@ -67,6 +67,8 @@ const (
 
 	HcpProduct        = "hcp"
 	HcpBillingAccount = "hcp-billing"
+
+	maxPositiveInt = 1<<63 - 1
```

Comment:
@thomasmckay What's the purpose of this definition and why is it set to this number?

Reply:
This is the largest int. When the size specified overflows in the k8s.io package, this is the value returned. Unfortunately the package doesn't throw an error in this case.

Reply:
Can we use `math.MaxInt` instead and I'd personally find your explanation useful as a comment if I was looking at this code?

Reply:
I should have looked harder for that const, thanks!